configs/sendbird.json

```diff
@@ -35,10 +35,11 @@
     "#change_log"
   ],
   "custom_settings": {
-    "separatorsToIndex": "_"
+    "separatorsToIndex": "_",
+    "attributesForFaceting": ["version"]
```

Comment:
Records extracted from your website do not have any `version` attribute. Please [add a meta tag within your webpage](https://community.algolia.com/docsearch/recommended-configuration.html#introduces-global-information-as-meta-tags) in order to let our crawler extract this information.

Reply:
Perfect, Merging and deploying